torch/_torch_docs.py

```diff
@@ -5000,13 +5000,14 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.linalg.matrix_power`
 """.format(**common_args))
 
-add_docstr(torch.matrix_exp,
-           r"""
+add_docstr(torch.matrix_exp, r"""
+matrix_exp(input) -> Tensor
+
 Returns the matrix exponential. Supports batched input.
```

Comment:
"Returns the matrix exponential. Supports batched input." -> "Computes the matrix exponential of a square matrix or of each square matrix in a batch."